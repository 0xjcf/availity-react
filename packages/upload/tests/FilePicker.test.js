import React from 'react';
import renderer from 'react-test-renderer';
import { FilePicker } from '../';

describe('Upload', () => {
  test('should render', () => {
    const component = renderer.create(<FilePicker onChange={() => {}} />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
