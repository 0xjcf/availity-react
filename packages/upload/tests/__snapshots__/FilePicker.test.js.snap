// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upload should render 1`] = `
<div
  className="custom-file"
>
  <input
    accept={undefined}
    className="custom-file-input"
    id="filePicker-1"
    onChange={[Function]}
    size={undefined}
    type="file"
    value=""
  />
  <label
    className="custom-file-label"
    htmlFor="filePicker-1"
  >
    Choose file
  </label>
</div>
`;
